code/game/objects/items/toys.dm

```diff
@@ -448,6 +448,13 @@
 	if(loc == user)
 		attack_self(user)
 
+/obj/item/toy/prize/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] is holding \the [src] above [user.p_them()]self and drops it on [user.p_their()] head! It looks like [user.p_theyre()] trying to commit suicide!</span>")
```

Comment:
```suggestion
	user.visible_message("<span class='suicide'>[src] fires its jump jets and hovers over [user]'s head! It looks like it's attempting Death From Above!</span>")
```

Reply:
Before you point it out, yes it's daft but ss13 was built on daft, please see the bag of holding suicide, the newspaper suicide and the box suicide for more information.